src/System.Private.CoreLib/shared/System/TimeSpan.cs

```diff
@@ -436,20 +436,11 @@ public bool TryFormat(Span<char> destination, out int charsWritten, ReadOnlySpan
             return new TimeSpan(-t._ticks);
         }
 
-        public static TimeSpan operator -(TimeSpan t1, TimeSpan t2)
-        {
-            return t1.Subtract(t2);
-        }
+        public static TimeSpan operator -(TimeSpan t1, TimeSpan t2) => t1.Subtract(t2);
 
-        public static TimeSpan operator +(TimeSpan t)
-        {
-            return t;
-        }
+        public static TimeSpan operator +(TimeSpan t) => t;
```

Comment:
not related to your PR: why this operator exists? how can I add a single `TimeSpan`?

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/arithmetic-operators#unary-plus-and-minus-operators